adding-behavior/changing-the-dom.md

```diff
@@ -0,0 +1,255 @@
+# Changing the DOM
+
+## Goals
+
+The goal for this lesson is to display one of JavaScript's strengths: accessing and changing the DOM.
+
+The skills of changing the DOM using vanilla JavaScript will transfer to other front-end skills, such as working with React.
```

Comment:
the vanilla javascript doesn't directly transfer, but other skills we develop while doing so do
```suggestion
The skills we develop when learning how to use vanilla JavaScript to change the DOM will transfer to other front-end skills, such as working with React.
```